test/logs/README.md

```diff
@@ -0,0 +1,39 @@
+## How to follow log outputs?
+
+- [How to follow PipelineRun logs?](#pipelinerun)
+- [How to follow TaskRun logs?](#taskrun)
+
+`Pipelinerun` object creates taskruns depending on the definition of underlying `Pipeline` definition.
+Each `TaskRun` object creates Kubernetes Pod and all the steps are translated into init containers.
+
+There is a "gotcha" about tailing logs for init container. Logs cannot be retrieved from pod that has been shut down for a while. In this case tailing logs will return error `Unable to retrieve container logs`.
+
+```shell
+go run test/logs/main.go [-n NAMESPACE] [[-pr PIPELINERUN-NAME] / [-tr TASKRUN_NAME]] [-f FILE-NAME]
```

Comment:
Either `-pr` or `-tr` is required right ? :thinking:

Reply:
Ah yes. Thanks for catching that